.codecov.yml

```diff
@@ -32,5 +32,3 @@ comment:
 
 ignore:
   - "src/nibetaseries/_version.py"
-  - "src/nibetaseries/conftest.py"
```

Comment:
I want these files to be included since if I have test code that is not being run, that is good to know